classes/PodsInit.php

```diff
@@ -150,6 +159,176 @@ public function plugins_loaded() {
 
 		load_plugin_textdomain( 'pods' );
 
+		$this->freemius();
+
+	}
+
+	/**
+	 * Handle Freemius SDK registration.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return \Freemius
+	 */
+	public function freemius() {
+		if ( $this->freemius ) {
+			return $this->freemius;
+		}
+
+		require_once dirname( __DIR__ ) . '/vendor/freemius/wordpress-sdk/start.php';
+
+		try {
+			$this->freemius = fs_dynamic_init( array(
+				'id'             => '5347',
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 77644cb87af2638f313c6bde899d9cf3</sub>